Sources/BraintreePayPalNativeCheckout/BTPayPalNativeOrderCreationClient.swift

```diff
@@ -28,18 +28,19 @@ class BTPayPalNativeOrderCreationClient {
     ) {
         apiClient.fetchOrReturnRemoteConfiguration { configuration, error in
             guard let config = configuration, error == nil else {
+                self.apiClient.sendAnalyticsEvent(BTPayPalNativeCheckoutAnalytics.orderCreationFailed)
```

Comment:
This fetch on line 29 is not fetching the orderID, but rather the merchant configuration.